spec/data_spec.rb

```diff
@@ -0,0 +1,25 @@
+require 'yaml'
+require 'spec_helper'
+
+RSpec.describe 'data' do
+  it 'has the same keys across all locales' do
+    keys = Dir['_data/*/settings.yml'].map { |path| hash_keys(YAML.load_file(path)) }
+
+    expect(keys).not_to be_empty
+    keys.combination(2).each do |first, second|
+      expect(first).to match_array(second)
+    end
```

Comment:
This usage of `Array#combination` threw me for a bit, our goal is to make sure they all match so what if we compare them all to the first environment? And throw an `aggregate_failure` around there in case there are multiple differences? We could adapt this so that the error message includes the locale key too if we preserved it from the path

```suggestion
    first_env_keys, *other_env_keys = keys

    aggregate_failures do
      other_env_keys.each do |other_keys|
        expect(first_env_keys).to match_array(other_keys)
      end
    end
```

Reply:
Yeah I think that'll work a bit better. Not as succinct, but technically we don't need to test every combination against each other if we're assuming they're all equal. I'll update.

Reply:
Updated in dda661f9, kinda clunky to maintain the locale detail in the error message, open to suggestions.